.gitignore

```diff
@@ -57,3 +57,4 @@ typings/
 # dotenv environment variables file
 .env
 
+*.sqlite3
```

Comment:
This isn't needed anymore

Reply:
_most_ of the gitignore isn't needed ;-)